package.json

```diff
@@ -8,7 +8,7 @@
     "clean": "lerna run clean --parallel",
     "test": "npm-run-all -p lint:check coverage test:size",
     "coverage": "jest --coverage --no-cache --ci --runInBand",
-    "lint:check": "eslint .",
+    "lint:check": "eslint . --fix",
```

Comment:
Please change this back. This is for CI to check that there are no linting errors.

Reply:
I will, debugging sometime else